### ISSUE TITLE
Touch target size of `Carousel` bullets doesn't meet WCAG 2.2 SC 2.5.8

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#4194]
### Contact Details

Slack: https://porschedev.slack.com/archives/CT7AVHTTQ/p1770808829378559

### JS Framework

all

### Porsche Design System version

3.32.0 / 4.0.0-alpha.0

### On which device(s) you can see the bug?

all

### In which browsers you can see the bug?

all

### Browser version

all

### Reproduction Example

Storefront

### Steps to reproduce the behavior

1. Go to https://designsystem.porsche.com/v3/components/carousel/configurator/
2. Click on bullets
3. See error: Touch target size doesn't meet WCAG 2.2 AA SC 2.5.8

### Expected behavior

Touch target size of clickable elements is at least 24x24px
